calculator.js

```diff
@@ -0,0 +1,26 @@
+const numOne = document.getElementById("num-one")
+const numTwo = document.getElementById("num-two")
+const equalsBtn = document.getElementById("equals-btn");
+const clearBtn = document.getElementById("clear");
+const digits = document.getElementById("digits");
+
+//Click event for the clear button
```

Comment:
Be aware of blank lines and make sure to remove notes to self when submitting code. "Code clean-up"